magic_spoiler/__main__.py

```diff
@@ -643,12 +644,13 @@ def main() -> None:
     # Cleanup outdated stuff that's not necessary
     changed |= delete_old_files()
 
-    # Set output to deploy
-    if changed:
-        print("::set-output name=deploy::true")
-    else:
-        print("::set-output name=deploy::false")
-
+    # Only when run in CI, enable deployment on changes
```

Comment:
this output is useful when testing, there is no reason to add this condition

Reply:
Yes, I do agree.

The GitHub Actions specific output is only omitted in their own CI logs, not when run locally! 👍